service/src/main/java/org/databiosphere/workspacedataservice/shared/model/job/JobStatus.java

```diff
@@ -1,11 +1,50 @@
 package org.databiosphere.workspacedataservice.shared.model.job;
 
+import org.databiosphere.workspacedataservice.generated.GenericJobServerModel;
+import org.springframework.http.HttpStatus;
+
 /** The various states a Job can move through. */
 public enum JobStatus {
-  QUEUED, // job has been created but not yet started
-  RUNNING, // job has started
-  SUCCEEDED, // job completed as expected
-  ERROR, // job failed
-  CANCELLED, // job was interrupted or intentionally not started
-  UNKNOWN
+  CREATED(HttpStatus.ACCEPTED, false, true), // job has been created but not yet scheduled to run
```

Comment:
These enums would benefit from the explainer comments I mentioned earlier too:

```suggestion
  CREATED(HttpStatus.ACCEPTED, /* isTerminated= */ false, /* isRunning=*/ true), // job has been created but not yet scheduled to run
```

Reply:
Huh - I get this for free for booleans in IntelliJ, I assumed it was the default:
![Screenshot 2023-10-12 at 3 12 24 PM](https://github.com/DataBiosphere/terra-workspace-data-service/assets/6041577/ec6b3b40-1198-41de-8d94-624623064319)